### DISCUSSION TITLE
[Feature] Automatic airline name matching based on flight number, for imported flights.

### DISCUSSION THREAD

**Discussion by jack2game** [johanohly/AirTrail#64] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm wondering if it is possible to have airline name automatically generated based on the flight number.

The reason I am looking for this is that after importing my App in the Air data, all the airline names are empty, and I have to manually add the airline for each of historical flights.

Some automation would be great. Thank you.

![chrome_1728574577_1TlZmpNmkb](https://github.com/user-attachments/assets/7db6dad8-f15b-438d-bf52-dfcd0588c748)

**Comment by johanohly** [comment]
Yes, I think that is a great idea!
I am on vacation for about two weeks though, so if you do not have a lot of flights, it would probably be a better idea to do it manually. If I implement this feature, you would also have to reimport your App in the Air data for it to take effect.

**Reply by jack2game** [reply]
Thank you. I have too many flights therefore will wait for your implementation and then re-import

**Comment by johanohly** [comment]
This has been added in v1.0.0!
